_posts/en/newsletters/2021-03-03-newsletter.md

```diff
@@ -0,0 +1,158 @@
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #16546][] introduces a new signer interface, allowing Bitcoin
+  Core to interact with external hardware signing devices through the
+  [HWI][topic hwi] or any other application which implements the same interface.
+
+    Bitcoin Core has been able to interface with hardware signers using HWI
+    [since Bitcoin Core version 0.18][hwi release]. Until this PR, however, [the
+    process][hwi old process] required use of the command line to transfer
+    data between Bitcoin Core and HWI. This PR simplifies the user experience
+    by enabling Bitcoin Core to directly communicate with HWI. The PR includes
+    [full documentation][hwi new process] on how to use the new signer interface
+    along with HWI.
+
+    The new signer interface is currently only accessible through RPC methods. A
+    [draft PR][signer gui] adds support for the signer interface to the GUI,
+    allowing the use of hardware signers with Bitcoin Core without any use of
+    the command line.
+
+- [Rust-Lightning #791][] adds support for polling `BlockSource` interfaces on
+  startup to sync blocks and headers, with fork detection during sync.
+  As described in [Newsletter #135][news135 blocksource], BlockSource
+  allows software to obtain data from sources beyond a standard Bitcoin
+  Core compatible node, allowing redundancy that can help prevent
+  [eclipse attacks][topic eclipse attacks] or other security problems.
+
+- [Rust-Lightning #794][] enables support for the [BOLT2][]
+  `option_shutdown_anysegwit` feature that permits future segwit versions when
+  initiating shutdown. If `option_shutdown_anysegwit` is negotiated, a channel
+  party sending a shutdown message to initiate closing may send a scriptpubkey
+  for payment, provided the script complies with the standard [BIP141][] witness
+  program form of a *version byte* (a 1-byte push opcode of `OP_1` through
+  `OP_16`) followed by a *witness program* (a byte vector push of 2 to 40
+  bytes). These shutdown scripts are limited to standard forms to avoid
+  expensive fee-heavy scripts or transactions with oversized scripts not
+  propagating due to non-standardness.  As it became [possible][0.19.0
+  segwit] to relay payments to any segwit script in Bitcoin Core
+  0.19.0.1 (released November 2019), it's now safe to [include
+  them][bolts #672] in LN's standard forms.
+
+- [HWI #413][], [#469][hwi #469], [#463][hwi #463], [#464][hwi #464],
+  [#471][hwi #471], [#468][hwi #468], and [#466][hwi #466] significantly
+  update and extend HWI's documentation.  Particularly notable changes
+  include a link to the documentation on [ReadTheDocs.io][hwi rtd], new
+  and updated [examples][hwi examples], and a new [policy][hwi policy]
+  that describes the criteria new devices must meet for HWI to consider
+  supporting them.
+
+- [Rust Bitcoin #573][] adds a new method
```

Comment:
Should this be "Rust-Bitcoin" instead of "Rust Bitcoin"?

Reply:
Going to leave as is for now as we have the project as Rust Bitcoin elsewhere. We can do a batch rename if needed.

Reply:
@jnewbery I chose "Rust Bitcoin" because that seems to be how they refer to themselves:

![2021-03-03-05_47_17_837273193](https://user-images.githubusercontent.com/61096/109831962-03b6be80-7be4-11eb-8277-8c1c4b4aca0f.png)

It's confusing because Matt's project is "Rust-Lightning" (which is how I refer to that):

![2021-03-03-05_49_00_417247110](https://user-images.githubusercontent.com/61096/109832277-4bd5e100-7be4-11eb-8ad4-85ba4ad4cbd4.png)

Reply:
Ah. I made the suggestion because it's referred to as "Rust-Bitcoin" later in the paragraph.

Reply:
Oh, that's a typo then.

Reply:
Fixed: https://github.com/bitcoinops/bitcoinops.github.io/pull/524